packages/ui/src/elements/Table/RelationshipProvider/index.tsx

```diff
@@ -62,20 +65,29 @@ export const RelationshipProvider: React.FC<{ readonly children?: React.ReactNod
           const url = `${serverURL}${api}/${slug}`
 
           const params = new URLSearchParams()
+          const select: SelectType = {}
 
           params.append('depth', '0')
           params.append('limit', '250')
 
+          const collection = collections.find((c) => c.slug === slug)
+          if (collection.admin.enableListViewSelectAPI) {
+            const fieldToSelect = collection.admin.useAsTitle ?? 'id'
+            select[fieldToSelect] = true
+
+            if (collection.upload) {
```

Comment:
@jhb-dev we could remove this line, the helper util already checks this condition internally.